sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -185,6 +190,7 @@ def test_use_okta_response_handler(self):
             key: value for key, value in source_config_details.items() if "proxy" not in key
         })
 
+        api_configuration.access_token = None
         api_configuration.certificate_filename = None
```

Comment:
Again, we need to exclude the new pat from the test.

Reply:
😃 
This is now fixed.
The previous change no longer required under revised approach.